ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -3123,6 +3123,24 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= []
       }
     )
+  ; ( "pool-start-ssh"
+    , {
+        reqd= []
+      ; optn= []
+      ; help= "Start and enable ssh service"
```

Comment:
I think the help message needs to show the difference with the host level API with the same name. E.g.  this is for all hosts in the pool.

Reply:
And it is also worth to mention that a new joined host in the pool or an ejected host from the pool would keep the original status.